src/deno_dir.rs

```diff
@@ -794,7 +794,8 @@ fn map_content_type(path: &Path, content_type: Option<&str>) -> msg::MediaType {
         "application/typescript"
         | "text/typescript"
         | "video/vnd.dlna.mpeg-tts"
-        | "video/mp2t" => msg::MediaType::TypeScript,
+        | "video/mp2t"
```

Comment:
"video/vnd.dlna.mpeg-tts" and "video/mp2t" are mine type for video file not typescript

Reply:
They are common types that are assigned by web servers to `.ts` files incorrectly. It is intentional.